Apps/phcheckpointfirewall/checkpoint_connector.py

```diff
@@ -486,6 +487,28 @@ def _unblock_ip(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS, "Successfully unblocked {0}".format('subnet' if length != '32' else 'IP'))
 
+    def _list_hosts(self, param):
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        if not(self._login(action_result)):
+            return action_result.get_status()
+
+        endpoint = 'show-hosts'
+
+        ret_val, resp_json = self._make_rest_call(endpoint, {}, action_result)
+
+        if ((not ret_val) and (not resp_json)):
+            return action_result.get_status()
+
+        action_result.add_data(resp_json)
+
+        total_num_hosts = resp_json.get('total')
```

Comment:
is this key guaranteed to exist? asking since it's doing a `.get` instead of using `[]` access. if it's not guaranteed do we need to handle this so a `None` value doesn't get propagated?

Reply:
correct @rgil-splunk! see changes below